extensions/amp-form/0.1/amp-form.js

```diff
@@ -346,67 +366,100 @@ export class AmpForm {
     return this.isCanonicalAction_ = parseUrl(url).origin == canonicalOrigin;
   }
 
+  handleXhrVerify_() {
+    if (this.state_ === FormState_.SUBMITTING) {
+      return;
+    }
+
+    this.setState_(FormState_.VERIFYING);
```

Comment:
Like blocking of the submit button? No, that is not planned. This state is intended to allow a spinner to show during the verify request. If the user wants to submit while the verify request is pending, they should be able to.